README.md

```diff
@@ -402,22 +475,64 @@ Example `process_exec` and `process_exit` events can be:
 </p>
 </details>
 
-For the rest of the use cases we will use the Tetragon CLI to give the output.
+#### Use case 2: Privileged Execution
+
+Tetragon also provides the ability to check process capabilities and kernel namespaces.
+
+As a first step let's enable visibility to capability and namespace changes via the configmap
+by setting `enable-process-cred` and `enable-process-ns`  from `false` to `true`
+```bash
+kubectl edit cm -n kube-system tetragon-config
+# change "enable-process-cred" from "false" to "true"
+# change "enable-process-ns" from "false" to "true"
+# then save and exit
+```
+Restart the Tetragon daemonset:
+```
+kubectl rollout restart -n kube-system ds/tetragon
+```
+
+As a second step, let's start monitoring the Security Observability events from the privileged `test-pod` workload:
+```bash
+kubectl logs -n kube-system -l app.kubernetes.io/name=tetragon -c export-stdout -f | tetra getevents --namespace default --pod test-pod
```

Comment:
Any chance we can use something else rather than test-pod? (e.g nginx?)

Reply:
Let's do it. I'd just use a simple nginx pod spec for this.

Reply:
See https://github.com/cilium/tetragon/issues/705